ado/ipacheckreadreplace.ado

```diff
@@ -110,21 +135,20 @@ qui {
 		drop if !missing(`selectmultiple')
 		save `allcorrections', replace
 		
+		******************* replace the selectmultiples ***********************
 		restore
-		* merge in the selectmultiples
 		tempvar mult_merge
-		merge 1:1 `id' using `selectmult', gen(`mult_merge')
+		merge 1:1 `id' using "`selectmult'", gen(`mult_merge')
 		count if `mult_merge' == 2
 		if `r(N)' > 0 {
 			noi di in red "`r(N)' corrections failed to merge"
 			noi list `id' if `mult_merge' == 2
 		}
 		drop if `mult_merge' == 2
 		drop `mult_merge'
-		
-		* replace the selectmultiples
+
 		tempvar concat
-		foreach var in `correction_vars' {
+		foreach var in `"`correction_vars'"' {
```

Comment:
This was part of my attempt to fix one of the odder errors I was getting, but it might not be necessary.